include/oneapi/tbb/task_group.h

```diff
@@ -99,7 +99,7 @@ class function_task : public task_handle_task  {
             next_task = successor_task;
         }
 #endif
-        finalize(&ed);
+        destroy(&ed);
```

Comment:
While not technically necessary, calling it as `this->destroy(&ed)` would make more obvious what is destroyed.

Reply:
Done